pkgs/top-level/all-packages.nix

```diff
@@ -17232,6 +17232,8 @@ with pkgs;
 
   ansible-doctor = with python3.pkgs; toPythonApplication ansible-doctor;
 
+  phpbench = callPackage ../development/tools/misc/phpbench { };
```

Comment:
If the php version needs to be overwritten, I think the better way is to do it here with `{ php = php80; }`

Reply:
Ok will do the change.